ops/lilypad-updater.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Detect your machine's architecture and set it as $OSARCH
+OSARCH=$(uname -m | awk '{if ($0 ~ /arm64|aarch64/) print "arm64"; else if ($0 ~ /x86_64|amd64/) print "amd64"; else print "unsupported_arch"}')
+export OSARCH
+
+# Detect your operating system and set it as $OSNAME
+OSNAME=$(uname -s | awk '{if ($1 == "Darwin") print "darwin"; else if ($1 == "Linux") print "linux"; else print "unsupported_os"}')
+export OSNAME
+
+# Get the latest version URL
+LATEST_URL=$(curl -s https://api.github.com/repos/lilypad-tech/lilypad/releases/latest | grep "browser_download_url.*lilypad-$OSNAME-$OSARCH" | cut -d : -f 2,3 | tr -d \")
```

Comment:
Does this check work with the new things added to binaries? (I mean the binaries with cuda and the ones without)

Reply:
We'll want to update this once we have the new releases pipeline in place:
```
grep "browser_download_url.*lilypad-$OSNAME-$OSARCH"
```
to

```
grep "browser_download_url.*lilypad-$OSNAME-$OSARCH-gpu"
```

Will add that to the releases pipeline PR if we merge this one first.